response_operations_ui/controllers/reporting_units_controllers.py

```diff
@@ -11,18 +12,21 @@
 logger = wrap_logger(logging.getLogger(__name__))
 
 
-def search_reporting_units(query):
+def search_reporting_units(query, limit, page):
+    start = datetime.datetime.utcnow()
```

Comment:
Need to remove debugging code

Reply:
Up to you guys , I left the timing in in case we wanted to splunk alert on elapsed time ?